syntax = "proto3";

package ImageComputer;

// The computeService service definition.
service ImageService {
  rpc GetImage (ImageRequest) returns (ImageResult) {}
}

message ImageRequest {
  int32 lightLevel = 1;
  int32 getNumbersOfImg = 2;
}

message ImageResult {
  int32 width = 1;
  int32 height = 2;
  bytes data = 3;
}
